packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -401,7 +401,16 @@ class _UiKitViewGestureRecognizer extends OneSequenceGestureRecognizer {
     team.captain = this;
     _gestureRecognizers = gestureRecognizerFactories.map(
       (Factory<OneSequenceGestureRecognizer> recognizerFactory) {
-        return recognizerFactory.constructor()..team = team;
+        final OneSequenceGestureRecognizer gestureRecognizer = recognizerFactory.constructor();
```

Comment:
Add a comment to explain here. Probably link to the comment that proposed the workaround.

Reply:
+1

Reply:
done